README.md

```diff
@@ -1,8 +1,200 @@
 # Auto-heal Service
 
-This project contains the _auto-heal_ service. It receives alert
-notitifications from the Prometheus alert manager and executes Ansible
-playbooks to resolve the root cause.
+This project contains the _auto-heal_ service. It receives alert notifications
+from the Prometheus alert manager and executes Ansible playbooks to resolve the
+root cause.
+
+## Configuration
+
+Most of the configuration of the auto-heal service is kept in a YAML
+configuration file.  The name of the configuration file is specified using the
+`-config-file` command line option. If this option isn't explicitly given then
```

Comment:
I am not 100% sure but I think that there is an openshift standard on using `--` as a long flag prefix: `--config-file`
@ironcladlou ?

Reply:
Should be using spf13/cobra like everything else

Reply:
I agree with using `spf13/cobra`, but this pull request is already very large. We will address this in a separate pull request. I created #7 to track that.